pvac_seq/main.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import os
+from subprocess import run, call, PIPE
+import sys
+
+current_directory = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+
+def run_pvac_script(script, *args):
+    pvac_cmd = "%s %s %s" % (
+        sys.executable,
+        os.path.join(current_directory, script),
+        " ".join(str(arg) for arg in args)
+    )
+    result = call([pvac_cmd], shell=True)
+    if(result):
+        print("Error while running:", script, args)
+        exit(result)
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("input",
+                        help="Input TSV File with variants (please provide complete path)"
+                        )
+    parser.add_argument("sample-name",
+                        help="Name of Sample; will be used as prefix for output files"
+                        )
+    parser.add_argument("netmhc-path",
+                        help="Path to local NetMHC3.4 installation"
+                        )
+    parser.add_argument("allele",
+                        help="Allele name to predict epitope prediction. Multiple alleles can be specified using a comma-separated list. For a list of available alleles, use: netMHC -A"
+                        )
+    parser.add_argument("epitope-length",
+                        help="length of subpeptides(epitopes) to predict ; Multiple lengths can be specified using a comma-separated list. Typical epitope lengths vary between 8-11."
+                        )
+    parser.add_argument("output-dir",
+                        help="Output directory for writing all result files"
+                        )
+    parser.add_argument("-l", "--variant-peptide-length",
+                        type=int,
+                        help="length of the peptide sequences in the input FASTA file; default 21",
+                        default=21,
+                        dest="peptide_sequence_length"
+                        )
+    parser.add_argument("-b","--binding-threshold",
+                        type=int,
+                        help="report only epitopes where the mutant allele has ic50 binding scores below this value ; default 500",
+                        default=500,
+                        dest="binding_threshold"
+                        )
+    parser.add_argument("-c", "--fold-change",
+                        type=int,
+                        help="Minimum fold change between mutant binding score and wild-type score. The default is 0, which filters no results, but 1 is often a sensible default (requiring that binding is better to the MT than WT)",
+                        default=0,
+                        dest="minimum_fold_change"
+                        )
+
+    args = parser.parse_args()
+    args.epitope_length = [int(epl) for epl in args.epitope_length.split(',')]
```

Comment:
Instead of doing the splitting here you can also set the `type` to `lambda s: [int(item) for item in s.split(',')]` in `add_argument` on line 34.